webpack.common.config.ts

```diff
@@ -70,12 +70,6 @@ module.exports = {
     creator_dashboard:
       commonPrefix + '/pages/creator-dashboard-page/' +
       'creator-dashboard-page.import.ts',
-    contributor_dashboard:
-      commonPrefix + '/pages/contributor-dashboard-page/' +
```

Comment:
Why are we removing contributor dashboard page?

Reply:
Unintentionally happened while resolving a merge conflict. Though this was already migrated to Angular but a recent PR #19811 reverted the migration. Reverted the changes here too. Done.